Casks/gpg-suite.rb

```diff
@@ -24,7 +24,7 @@
 
   uninstall_postflight do
     ["gpg", "gpg2", "gpg-agent"].map { |exec_name| "/usr/local/bin/#{exec_name}" }.each do |exec|
-      FileUtils.rm_rf(exec) && File.readlink(exec).include?("MacGPG2")
+      File.readlink(exec).include?("MacGPG2") && FileUtils.rm_rf(exec)
```

Comment:
```suggestion
      next unless File.exist?(exec)
      next unless File.readlink(exec).include?("MacGPG2")

      FileUtils.rm_rf(exec)
```
I'm tempted to make `/usr/local/bin` a `Pathname` so we don't need to keep using the `File` class.

Reply:
So something like:
```
["gpg", "gpg2", "gpg-agent"].map { |exec_name| Pathname.new("/usr/local/bin/#{exec_name}") }.each do |exec|
    next unless exec.exist?
    next unless exec.readlink.to_s.include?("MacGPG2")

    exec.unlink
end
```
?

Reply:
Sure, or even

```ruby
["gpg", "gpg2", "gpg-agent"].map { |exec_name| Pathname("/usr/local/bin")/exec_name }.each do |exec|
    next unless exec.exist?
    next unless exec.readlink.to_s.include?("MacGPG2")

    exec.unlink
end
```